Source/BSGReachability.h

```diff
@@ -0,0 +1,102 @@
+/*
+ Copyright (c) 2011, Tony Million.
```

Comment:
Vendoring this in complicates the license. I'm also not entirely sure the license for this version of Reachability is valid given that its derived from Apple code. We could depend on it by adding it to our podspec/Cartfile/submodules, but my overall perception is that for what we need, it might be overkill.